tests/epyccel/test_epyccel_variable_annotations.py

```diff
@@ -166,3 +166,20 @@ def homogeneous_tuple_annotation():
 
     assert epyc_homogeneous_tuple_annotation() == homogeneous_tuple_annotation()
     assert isinstance(epyc_homogeneous_tuple_annotation(), type(homogeneous_tuple_annotation()))
+
+@pytest.mark.parametrize( 'language', [
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("fortran", marks = [
+            pytest.mark.skip(reason="append() not implemented in fortran"),
+            pytest.mark.fortran]),
+        pytest.param("python", marks = pytest.mark.python)
+    ]
+)
+
+def test_homogeneous_set_annotation_int(language):
+    def homogeneous_set_annotation ():
+        a : set[int]
```

Comment:
You will have to temporarily tell pylint to ignore this error:
```suggestion
        a : set[int] #pylint: disable=unused-variable
```

Reply:
fixed e87dbf113f01d210f72bcc5679bddff399985988